Add Sliding Rail Stop Ponder content 添加滑轨站的思索内容

ForgeStove opened the pull request:
为滑轨站(Sliding Rail Stop)添加了基本的思索内容
优化了活塞思索动画
修正了溜槽在思索内的方向问题

ForgeStove committed:
修复滑轨动画问题

ForgeStove committed:
修复 Anvil-Dev/AnvilCraft#2600 的滑轨动画问题

ForgeStove committed:
添加滑轨站的思索，优化部分其他思索

ForgeStove committed:
Merge remote-tracking branch 'origin/releases/1.21' into releases/1.21

# Conflicts:
#	src/main/java/dev/dubhe/anvilcraft/integration/ponder/scene/logistics/SlidingRailScene.java

ForgeStove committed:
添加滑轨站的思索文本

ForgeStove committed:
Merge branch 'Anvil-Dev:releases/1.21' into releases/1.21

ForgeStove committed:
更新思索

TBPig commented on src/main/java/dev/dubhe/anvilcraft/integration/ponder/AnvilCraftPonderScenes.java:
```diff
@@ -1,9 +1,7 @@
-import dev.dubhe.anvilcraft.integration.ponder.scene.logistics.ChuteScene;
-import dev.dubhe.anvilcraft.integration.ponder.scene.logistics.MagneticChuteScene;
-import dev.dubhe.anvilcraft.integration.ponder.scene.logistics.SlidingRailScene;
+import dev.dubhe.anvilcraft.integration.ponder.scene.logistics.*;
```
出现这个问题可以导入“贡献指南”下的代码格式模板

ForgeStove committed:
修正style

ForgeStove committed:
Merge branch 'Anvil-Dev:releases/1.21' into releases/1.21

Gu-ZT committed:
Merge remote-tracking branch 'upstream/releases/1.21' into pr/2608

Gu-ZT committed:
docs(ponder): 更新滑动轨道相关文档

- 新增带电滑动轨道的文档，说明其基本功能和使用方法
- 新增滑动轨道停止器的文档，介绍其吸引和停止实体的能力
- 更新普通滑动轨道的文档，补充粘液块在轨道上的应用
- 移除多块滑动轨道的冗余说明
